libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md

```diff
@@ -0,0 +1,76 @@
+# Sierra-F1 Autopilot by Sierra Aerospace
+https://www.sierraaerospace.in/
+
+## Features
+# Main Board:
+ - STM32H743 microcontroller
+ - 32MB QSPI flash
+ - BMI088+ICM42688 IMUs
+ - DPS310 barometer on SPI 
+ - SDMMC support
+ - 4 UARTs- 2 with HW flow control
+ - 6 PWM outputs with DSHOT capability
+ - Dual FDCAN
+ - RCIN port
+ - Dual power input with 2xI2C
+ - No co-processor
+
+# Expansion board:
+ -Ethernet PHY
+ -ICM42688/ICM45686/ADIS16507
+ -DPS310
+ -More PWMs with ALT function
```

Comment:
I assume this is a different expansion board....but I dont see how it will work since you do not specify and outputs for it in the hwdef.dat....please explain....

Reply:
Currently it's Ethernet+IMU+Baro. There's plan to have combination of sensors in future including more PWMs.